lib/ansible/modules/cloud/amazon/rds.py

```diff
@@ -584,16 +584,69 @@ def get_data(self):
         d = {
             'id': self.name,
             'create_time': self.instance['InstanceCreateTime'],
+            'engine': self.instance['Engine'],
```

Comment:
Can you add [RETURN docs](https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#return-block) please? There is also a pull request out to [separate rds functionality](https://github.com/ansible/ansible/pull/25646); if there is return info you think should be added please feel free to leave some feedback.